### ISSUE TITLE
Is this only works for Feed Forward?

### ISSUE THREAD

**Issue by ceste** [surmenok/keras_lr_finder#1]
Hi, 

Thank you for providing this awesome library. I gave it a try to my CNN model and it did not work well. Is this only works for Feed Forward model?

Please advise

**Comment by surmenok** [comment]
No, it should work for different kinds of neural networks, including CNN. This approach was tried on fully connected networks, CNN and LSTM.

**Comment by tarasivashchuk** [comment]
This will work for Model object (keras.models.Model or tf.keras.models.Model) as long as the optimizer function used when compiling the model has a "learning_rate" parameter.

 Actually, I just checked the code for this repo and it uses "lr", which is technically the same in the Keras source code, but only supported for the sake of backwards compatibility. The recommended usage "learning_rate", and I will refactor that and make a pull request after I'm done with this comment. 😄

So you could even a multi-input model with a CNN, an RNN, and a Feed-Forward network within one model and use this - the model doesn't matter. I would recommend learning a bit more about how this works and the corresponding login.

Here are some resources you could look over:
["How to Use the Learning Rate Finder in TensorFlow"](https://medium.com/octavian-ai/how-to-use-the-learning-rate-finder-in-tensorflow-126210de9489) by [Ashwath Salimath](https://medium.com/@ashwaths?source=post_page-----126210de9489----------------------)
[Learning Rate Finder Documentation from Fast.ai](https://docs.fast.ai/callbacks.lr_finder.html)

Also, I'm not sure how, but this issue should be closed.
